DependencyInjection/HWIOAuthExtension.php

```diff
@@ -131,22 +131,32 @@ public function load(array $configs, ContainerBuilder $container)
      */
     public function createResourceOwnerService(ContainerBuilder $container, $name, array $options)
     {
-        // alias services
+        // Decorate the given service if one is specified
         if (isset($options['service'])) {
-            // set the appropriate name for aliased services, compiler pass depends on it
-            $container->setAlias('hwi_oauth.resource_owner.'.$name, $options['service']);
+            $definition = $container->findDefinition($options['service']);
```

Comment:
This will never work. The DI extension cannot access services defined by other bundles